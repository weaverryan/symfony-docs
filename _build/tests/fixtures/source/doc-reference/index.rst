A test for doc link
===================

:doc:`A doc test </file>`

.. toctree::
    :hidden:

    file
