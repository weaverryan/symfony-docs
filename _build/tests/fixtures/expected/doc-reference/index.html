<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="section" id="a-test-for-doc-link">
    <h1>A test for doc link<a class="headerlink" href="#a-test-for-doc-link" title="Permalink to this headline">Â¶</a></h1>
    <p><a href="file.html" class="reference internal"><em>A doc test</em></a></p>
</div>
</body>
</html>
